packages/core/src/data-grid/data-grid.tsx

```diff
@@ -283,7 +283,7 @@ type DamageUpdateList = readonly {
 
 export interface DataGridRef {
     focus: () => void;
-    getBounds: (col: number, row?: number) => Rectangle | undefined;
+    getBounds: (col?: number, row?: number) => Rectangle | undefined;
```

Comment:
oh i see this is shared... hmmmmm

Reply:
yep, the interface is shared 👍  So both need to somehow handle `col===undefined`. We could also return `undefined` in `getBoundsForItem` if `col` is `undefined`.